cards.py

```diff
@@ -46,7 +46,7 @@ def get(self, id):
 			return
 		
 		# TODO: Replace with fetching the current semester once semester formatting functions are merged in.
-		semester = 'Spring 2017'
+		semester = 'Fall 2017'
```

Comment:
Don't we have semester formatting functions in `semesters.py` now?

Reply:
Yes, but I want to save integrating them into the ID card generator for a future release to minimize the amount of testing needed for this one.

Reply:
Sure. Looks good to merge